src/main/java/woowacourse/shoppingcart/domain/cart/Cart.java

```diff
@@ -0,0 +1,57 @@
+package woowacourse.shoppingcart.domain.cart;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import woowacourse.shoppingcart.exception.InvalidCartItemException;
+
+public class Cart {
+
+    private final List<CartItem> value;
```

Comment:
```suggestion
    private final List<CartItem> cartItems;
```

더 명확한 이름이 좋아 보이네요 :)

Reply:
넵 저도 위 상황에서는 `values` 정도로 사용하고 더 명확하게 처리하고 싶을 땐 `{element}s`로 사용하고 있어요. :)